src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/Jobs/JobsService.cs

```diff
@@ -23,7 +23,7 @@ public async Task RefreshLegalEntities(int pageNumber, int pageSize)
             };
 
             var response = await _client.PutAsJsonAsync(
-                "/jobs", 
```

Comment:
Would it not be easier just to take the / from the front of /jobs?

Reply:
I tried that abut it didn't fix the issue.

Reply:
If there is no AbsolutePath then _client.BaseAddress.AbsolutePath returns "/"

Reply:
You are correct.  The slash can be removed but only if it is added to the base address!